File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    n = 0
    for i in s:
        if i == " ":
            answer += " "
            n = 0
        else:
            if n%2 == 0:
                answer += i.upper()
                n +=1
            else:
                answer += i.lower()
                n +=1
    return answer

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.kt
class Solution {
    fun solution(s: String): String {
        var answer = ""
        val l = s.split(" ")
        val total = l.size
        var count = 0
        for(i in l){
            count +=1
            val n = i.length
            for(j in 0..n-1){
                if (j%2 == 0) {
                    answer += i[j].toUpperCase()
                }
                else{
                    answer += i[j].toLowerCase()
                }
            }
            if (count != total){
                answer += " "
            }
        }
        return answer
    }
}